src/labelle/lib/devices/device_config.py

```diff
@@ -18,30 +18,35 @@ class DeviceConfig:
     printHeadSizePixels: int
     """Size of the print head in pixels (use calibration routine to determine)"""
 
+    printHeadActiveAreaMillimeters: float
+    """Size of the active area of the print head in millimters
+        (use calibration routine to determine)"""
+
     supportedTapeSizes: list[int]
     """List of supported tape sizes in mm"""
 
-    marginsPerTape: dict[int, Tuple[int, int]]
-    """
-    Dictonary of print margins per tape size
-    Entry: TapeSizeMM : (topMarginInPx, bottomMarginInPx)
-    Use the calibration routine to determine these for each tape size
-    """
+    # Fixed to 1 mm until proven otherwise ;)
+    TAPE_ALIGNMENT_INACCURACY_MM: float = 1
+    """The inaccuracy of the tape position relative to the printhead"""
+    # Inaccuracy of the tape position is mostly caused by
+    # the tape moving slightly from side to side in the cartridge.
+    # Improper cartrigde placemement is also an factor,
+    # but negligible due to a physical spring in the lid.
```

Comment:
Please make lines longer.
I think you should be able to write all comments under a single `""" .... """` block, just make sure to add the extended description following a separating empty line. https://stackoverflow.com/a/23188939/1692287

Reply:
I want to make long lines, but there is a pre-commit hook from preventing me to...

I will add the extended description to the function comment.

Reply:
```python
    """The inaccuracy of the tape position relative to the printhead

    Inaccuracy of the tape position is mostly caused by
    the tape moving slightly from side to side in the cartridge.
    Improper cartrigde placemement is also an factor,
    but negligible due to a physical spring in the lid.
    """
```

Reply:
The purpose of the pre-commit hook is not to make things pretty. Instead the idea is to reduce entropy, so that there are fewer ways to express equivalent code. (Even if it's uglier, at least it's more standardized.)

If it gets in your way, feel free to ignore the errors for now and use `git commit --no-verify`, and we can deal with it once at the end.

Reply:
Fixed as suggested.